### PR TITLE
prefer python 3

### PR DESCRIPTION
**Pull request by philandstuff**
document that we prefer python 3 over python 2.

### CODE REVIEW

**Comment by philandstuff** [comment]
this fixes #52

File: source/standards/programming-languages.html.md.erb [hunk 1]
```diff
@@ -31,6 +31,16 @@ teams at the moment, and we are confident in how to host and operate
 applications written in them. We'd advise you to carry out new
 development in one of them.
 
+### Python
+
+New python projects should be in python 3.  Python 2 will
+[reach end of life in 2020][PEP373].  Python 3 is now well-supported
+by application frameworks and libraries, and is commonly used in
+production.  There is no longer any reason not to prefer python 3 over
```
Comment (andreagrandi) [inline source/standards/programming-languages.html.md.erb]: I would just change "python" to "Python", but other than this it's fine!

**Comment by Wynndow** [comment]
Digital Marketplace are currently in the process of upgrading our apps from Python 2 to Python 3. Our API is already there, the remaining apps should follow in this sprint.

**Comment by andreagrandi** [comment]
@timblair hi! I'm from "Better Use of Data", how can I help?

File: source/standards/programming-languages.html.md.erb [hunk 2]
```diff
@@ -31,6 +31,16 @@ teams at the moment, and we are confident in how to host and operate
 applications written in them. We'd advise you to carry out new
 development in one of them.
 
+### Python
+
+New Python projects should be in Python 3.  Python 2 will
+[reach end of life in 2020][PEP373].  Python 3 is now well-supported
+by application frameworks and libraries, and is commonly used in
+production.  There is no longer any reason not to prefer Python 3 over
```
Comment (davbo) [inline source/standards/programming-languages.html.md.erb]: Could we remove this final sentence? It's a double negative and I think what's there already does a good job at making the point.

**Comment by philandstuff** [comment]
I spoke to Dan Heron from better use of data and he says they're happy with this wording.

**Pull request merged**